aten/src/ATen/native/TensorShape.cpp

```diff
@@ -2880,7 +2890,8 @@ Tensor squeeze_qtensor(const Tensor& self, c10::optional<int64_t> dim) {
                                                   axis,
                                                   quantizer->scalar_type());
   }
-  auto result = make_qtensor(self, sizes, strides, quantizer);
+  // quantized kernels shouldn't run with symints
```

Comment:
actually, eventually, they should, but we don't have enough basic support for it yet